From ba292fd6c7e45fa658b1e6d3f66cc474834dd5d2 Mon Sep 17 00:00:00 2001
From: Andy Yan <andy.yan@rock-chips.com>
Date: Tue, 17 Jul 2018 19:59:13 +0800
Subject: [PATCH 47/73] arm: configs: rk3308_linux_aarch32_defconfig: sync from
 develop-4.4

Sync from rockchip develop-4.4 41c170c42fed("scripts: make
gcc-wrapper.py compatible with python 2.7 and 3")

Add more feature:
CONFIG_CGROUPS=y
CONFIG_CFG80211=y
CONFIG_VT=y

Change-Id: I4fa18e60b441ea6824bd207e613c45becbcadfa5
Signed-off-by: Andy Yan <andy.yan@rock-chips.com>
---
 arch/arm/configs/rk3308_linux_aarch32_defconfig | 24 +++++++++++++-----------
 1 file changed, 13 insertions(+), 11 deletions(-)

diff --git a/arch/arm/configs/rk3308_linux_aarch32_defconfig b/arch/arm/configs/rk3308_linux_aarch32_defconfig
index db7c1460a8a2..1fc6a2c2b91c 100644
--- a/arch/arm/configs/rk3308_linux_aarch32_defconfig
+++ b/arch/arm/configs/rk3308_linux_aarch32_defconfig
@@ -5,8 +5,14 @@ CONFIG_DEFAULT_HOSTNAME="localhost"
 CONFIG_SYSVIPC=y
 CONFIG_NO_HZ=y
 CONFIG_HIGH_RES_TIMERS=y
-CONFIG_LOG_BUF_SHIFT=18
+CONFIG_LOG_CPU_MAX_BUF_SHIFT=0
 CONFIG_CGROUPS=y
+CONFIG_BLK_DEV_INITRD=y
+# CONFIG_RD_BZIP2 is not set
+# CONFIG_RD_LZMA is not set
+# CONFIG_RD_XZ is not set
+# CONFIG_RD_LZO is not set
+# CONFIG_RD_LZ4 is not set
 CONFIG_CC_OPTIMIZE_FOR_SIZE=y
 # CONFIG_BASE_FULL is not set
 CONFIG_EMBEDDED=y
@@ -33,7 +39,7 @@ CONFIG_AEABI=y
 # CONFIG_COMPACTION is not set
 CONFIG_DEFAULT_MMAP_MIN_ADDR=32768
 CONFIG_UACCESS_WITH_MEMCPY=y
-CONFIG_CMDLINE="coherent_pool=0 user_debug=31"
+CONFIG_CMDLINE="coherent_pool=0 loglevel=0"
 CONFIG_CMDLINE_EXTEND=y
 CONFIG_CPU_FREQ=y
 CONFIG_CPU_FREQ_GOV_USERSPACE=y
@@ -66,7 +72,6 @@ CONFIG_BT_HCIUART=y
 CONFIG_BT_HCIUART_H4=y
 CONFIG_BT_HCIUART_LL=y
 CONFIG_CFG80211=y
-CONFIG_MAC80211=y
 CONFIG_RFKILL=y
 CONFIG_DEVTMPFS=y
 CONFIG_DEVTMPFS_MOUNT=y
@@ -120,7 +125,6 @@ CONFIG_CPU_THERMAL=y
 CONFIG_DEVFREQ_THERMAL=y
 CONFIG_ROCKCHIP_THERMAL=y
 CONFIG_WATCHDOG=y
-CONFIG_SOFT_WATCHDOG=y
 CONFIG_DW_WATCHDOG=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
 CONFIG_REGULATOR_GPIO=y
@@ -144,11 +148,11 @@ CONFIG_SND_SOC_DUMMY_CODEC=y
 CONFIG_SND_SOC_RK3308=y
 CONFIG_SND_SIMPLE_CARD=y
 # CONFIG_HID_GENERIC is not set
-CONFIG_USB_DWC2=y
-CONFIG_USB_GADGET=y
+CONFIG_USB_DWC2=m
+CONFIG_USB_GADGET=m
 CONFIG_USB_GADGET_DEBUG_FILES=y
 CONFIG_USB_GADGET_VBUS_DRAW=500
-CONFIG_USB_CONFIGFS=y
+CONFIG_USB_CONFIGFS=m
 CONFIG_USB_CONFIGFS_F_FS=y
 CONFIG_MMC=y
 CONFIG_MMC_BLOCK_MINORS=32
@@ -175,7 +179,7 @@ CONFIG_FIQ_DEBUGGER_CONSOLE_DEFAULT_ENABLE=y
 CONFIG_RK_CONSOLE_THREAD=y
 CONFIG_FIQ_DEBUGGER_TRUST_ZONE=y
 # CONFIG_ARM_ARCH_TIMER_EVTSTREAM is not set
-CONFIG_ROCKCHIP_IOMMU=y
+# CONFIG_IOMMU_SUPPORT is not set
 # CONFIG_CPU_RK312X is not set
 # CONFIG_CPU_RK3036 is not set
 # CONFIG_CPU_RK30XX is not set
@@ -195,7 +199,7 @@ CONFIG_IIO_TRIGGER=y
 CONFIG_ROCKCHIP_SARADC=y
 CONFIG_PWM=y
 CONFIG_PWM_ROCKCHIP=y
-CONFIG_PHY_ROCKCHIP_INNO_USB2=y
+CONFIG_PHY_ROCKCHIP_INNO_USB2=m
 CONFIG_ANDROID=y
 CONFIG_NVMEM=y
 CONFIG_RK_FLASH=y
@@ -217,7 +221,6 @@ CONFIG_PSTORE_RAM=y
 # CONFIG_NETWORK_FILESYSTEMS is not set
 CONFIG_PRINTK_TIME=y
 CONFIG_DEBUG_INFO=y
-CONFIG_DEBUG_FS=y
 CONFIG_MAGIC_SYSRQ=y
 CONFIG_MAGIC_SYSRQ_DEFAULT_ENABLE=0
 CONFIG_LOCKUP_DETECTOR=y
@@ -230,7 +233,6 @@ CONFIG_PANIC_TIMEOUT=1
 CONFIG_RCU_CPU_STALL_TIMEOUT=60
 # CONFIG_FTRACE is not set
 CONFIG_STRICT_DEVMEM=y
-CONFIG_DEBUG_USER=y
 CONFIG_DEBUG_SET_MODULE_RONX=y
 # CONFIG_CRYPTO_ECHAINIV is not set
 # CONFIG_CRYPTO_HW is not set
-- 
2.11.0

